app/models/comment.rb

```diff
@@ -0,0 +1,15 @@
+class Comment < ApplicationRecord
+  belongs_to :debate
+  belongs_to :slack_user, class_name: Slack::User, foreign_key: :slack_user_id, optional: true
+  validates :content, presence: true
+
+  def user
+    slack_user || anonymous_user
+  end
+
+  private
+
+  def anonymous_user
+    OpenStruct.new(image_url: '/assets/dashboard/default_user.png', name: 'Anonymous')
```

Comment:
you could cache it on a class level because it is constant. you can move it to class method and cache using class instance variable